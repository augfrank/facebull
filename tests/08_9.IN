M0   C0   C1  2717
M1   C0   C2  2180
M2   C0   C3  2449
M3   C0   C4  2722
M4   C0   C5  2330
M5   C0   C6  2408
M6   C0   C7  2010
M7   C1   C0  2814
M8   C1   C2  2411
M9   C1   C3  2034
M10   C1   C4  2700
M11   C1   C5  2012
M12   C1   C6  2688
M13   C1   C7  2394
M14   C2   C0  2110
M15   C2   C1  2666
M16   C2   C3  2735
M17   C2   C4  2353
M18   C2   C5  2337
M19   C2   C6  2920
M20   C2   C7  2994
M21   C3   C0  2666
M22   C3   C1  2421
M23   C3   C2  2705
M24   C3   C4  2426
M25   C3   C5  2954
M26   C3   C6  2404
M27   C3   C7  2233
M28   C4   C0  2308
M29   C4   C1  2730
M30   C4   C2  2935
M31   C4   C3  2418
M32   C4   C5  2029
M33   C4   C6  2378
M34   C4   C7  2135
M35   C5   C0  2211
M36   C5   C1  2268
M37   C5   C2  2733
M38   C5   C3  2913
M39   C5   C4  2152
M40   C5   C6  2316
M41   C5   C7  2082
M42   C6   C0  2413
M43   C6   C1  2257
M44   C6   C2  2454
M45   C6   C3  2444
M46   C6   C4  2541
M47   C6   C5  2365
M48   C6   C7  2002
M49   C7   C0  2245
M50   C7   C1  2549
M51   C7   C2  2517
M52   C7   C3  2792
M53   C7   C4  2310
M54   C7   C5  2305
M55   C7   C6  2749
