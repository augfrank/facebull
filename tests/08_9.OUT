17515
1 9 17 26 32 36 48 49
